lib/engine/game/g_system18/map_gotland_customization.rb

```diff
@@ -137,7 +140,54 @@ def map_gotland_game_trains(trains)
         end
 
         def map_gotland_game_phases
-          self.class::S18_INCCAP_PHASES
+          [
+            {
+              name: '2',
+              train_limit: 3,
+              tiles: [:yellow],
+              operating_rounds: 2,
+            },
+            {
+              name: '3',
+              on: '3',
+              train_limit: 3,
+              tiles: %i[yellow green],
+              operating_rounds: 2,
+              status: ['nr_or'],
+            },
+            {
+              name: '4',
+              on: '4',
+              train_limit: 3,
+              tiles: %i[yellow green],
+              operating_rounds: 2,
+              status: ['nr_or'],
+            },
+            {
+              name: '5',
+              on: '5',
+              train_limit: 2,
+              tiles: %i[yellow green brown],
+              operating_rounds: 2,
+              status: ['nr_or'],
+            },
+            {
+              name: '6',
+              on: '6',
+              train_limit: 2,
+              tiles: %i[yellow green brown],
+              operating_rounds: 2,
+              status: ['nr_or'],
+            },
+            {
+              name: '8',
+              on: '8',
+              train_limit: 2,
+              tiles: %i[yellow green brown gray],
+              operating_rounds: 2,
+              status: ['nr_or'],
+            },
+          ].deep_freeze
```

Comment:
As an alternative to essentially redefining S18_INCCAP_PHASES just to add status lines, you could just add the status to post_game_phases. See Poland for an example.

Reply:
Since you approved this I'm gonna merge, and this could be changed in a follow-up if y'all want to.